packages/labs/virtualizer/src/support/resize-observer-errors.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {
+  interceptMethod,
+  MethodInterceptorTeardown,
+} from './method-interception.js';
+
+/**
+ * In a testing environment with a setup/before and teardown/after callback pattern, this function
+ * can be used to quickly install ignore benign errors that are expected to occur in tests.  For example, if a
```

Comment:
unfinished comment?

Reply:
Oh goodness! Thanks for catching that.  I had an unsaved file in my editor I totally missed that.  Fixed; including param descriptions for before and after.